search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/GenerativeQAResponseProcessor.java

```diff
@@ -115,6 +132,12 @@ private List<String> getSearchResults(SearchResponse response) {
         return searchResults;
     }
 
+    private static String jsonArrayToString(List<String> listOfStrings) {
+        JsonArray array = new JsonArray(listOfStrings.size());
+        listOfStrings.forEach(array::add);
```

Comment:
do we need this line?

Reply:
This is used to construct a single string out of an array as a helper method to store the search results as a "additionalInfo" string in memory.